Source/WebCore/dom/Node.h

```diff
@@ -579,8 +582,9 @@ class Node : public EventTarget {
         IsComputedStyleInvalidFlag = 1 << 25,
         HasShadowRootContainingSlots = 1 << 26,
         IsInTopLayer = 1 << 27,
+        NeedsSVGRendererUpdate = 1 << 28
```

Comment:
I suppose this flag can only be set on SVGElements so could be there (but there are free bits so it doesn't really matter much).

Reply:
I'm going to leave it here since SVGElement doesn't have its own flags word currently.